proprietary/Components/src/utrecht/form-field-description.tokens.json

```diff
@@ -0,0 +1,12 @@
+{
+  "utrecht": {
+    "form-field": {
+      "description": {
+        "color": { "value": "{denhaag.color.grey.4}" },
+        "font-family": { "value": "{utrecht.document.font-family}" },
+        "font-size": { "value": "{denhaag.typography.scale.s.font-size}" },
+        "font-weight": { "value": "{denhaag.typography.weight.regular}" }
```

Comment:
Instead of the `"error"` color token in denhaag/ you can specify the `"invalid": { "color": ` token here.

See: https://github.com/nl-design-system/utrecht/blob/main/components/form-field-description/tokens.json#L47